SQL/tgstation_schema_prefixed.sql

```diff
@@ -31,7 +31,7 @@ CREATE TABLE `SS13_admin` (
 -- Table structure for table `SS13_admin_log`
 --
 
-DROP TABLE IF EXISTS `SS13_dmin_log`;
+DROP TABLE IF EXISTS `SS13_admin_log`;
```

Comment:
HA

Reply:
Maybe we should leave it just so nobody tries to use this system.